File: src/main/java/no/fint/organization/OrganizationRepository.java
package no.fint.organization;

import org.springframework.data.mongodb.repository.MongoRepository;
import org.springframework.stereotype.Repository;

import java.util.List;

@Repository
public interface OrganizationRepository extends MongoRepository<OrganizationDocument, String> {

    List<OrganizationDocument> getAllByOrgId(String orgid);
}

File: src/main/kotlin/no/fint/organization/OrganizationRepository.kt
package no.fint.organization

import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface OrganizationRepository : JpaRepository<OrganizationDocument, String> {
    fun getAllByOrgId(orgId: String): List<OrganizationDocument>

    fun getOrganizationDocumentByIdAndOrgId(
        id: String,
        orgId: String,
    ): OrganizationDocument?
}
